src/pevm.rs

```diff
@@ -328,6 +262,54 @@ pub fn execute_revm_sequential<S: Storage>(
     Ok(results)
 }
 
+fn canonize_account(
+    results: &mut [PevmTxExecutionResult],
+    address: Address,
+    initial_account_basic: AccountBasic,
```

Comment:
We can pass in `storage` and calculate `current_account` inside here with the same formula for everyone?